hu/server/users.php

```diff
@@ -2,32 +2,32 @@
+        'control_console' => 'Lehetővé teszi a felhasználó számára, hogy adatokat küldjön a szerver konzoljának.',
+        'control_start' => 'Lehetővé teszi a felhasználó számára a szerver elindítását.',
+        'control_stop' => 'Lehetővé teszi a felhasználó számára a szerver leállítását.',
+        'control_restart' => 'Lehetővé teszi a felhasználó számára a szerver újraindítását.',
+        'control_kill' => 'Lehetővé teszi a felhasználó számára a szerver megölését.',
+        'user_create' => 'Lehetővé teszi a felhasználó számára, hogy új felhasználói fiókokat hozzon létre a szerverhez.',
+        'user_read' => 'Engedélyezi a felhasználó számára a szerverhez társított felhasználók megtekintését.',
+        'user_update' => 'Lehetővé teszi a felhasználó számára, hogy módosítsa a szerverhez társított más felhasználókat.',
+        'user_delete' => 'Lehetővé teszi a felhasználó számára a szerverhez társított többi felhasználó törlését.',
+        'file_create' => 'Engedélyezi a felhasználó számára új fájlok és könyvtárak létrehozását.',
+        'file_read' => 'Lehetővé teszi a felhasználó számára a szerverhez társított fájlok és mappák, valamint azok tartalmának megtekintését.',
+        'file_update' => 'Lehetővé teszi a felhasználó számára, hogy frissítse a szerverhez társított fájlokat és mappákat.',
+        'file_delete' => 'Lehetővé teszi a felhasználó számára fájlok és könyvtárak törlését.',
+        'file_archive' => 'Lehetővé teszi a felhasználó számára fájlarchívumok létrehozását és a meglévő archívumok kibontását.',
+        'file_sftp' => 'Lehetővé teszi a felhasználó számára, hogy a fenti fájlműveleteket SFTP kliens segítségével hajtsa végre.',
+        'allocation_read' => 'Hozzáférést engedélyez a szerver kiosztást kezelő oldalakhoz.',
+        'allocation_update' => 'Engedélyezi a felhasználó számára, hogy módosítsa a szerver kiosztásait.',
+        'database_create' => 'Engedélyezi a felhasználó számára, hogy új adatbázist hozzon létre a szerverhez.',
+        'database_read' => 'Engedélyezi a felhasználó számára a szerver adatbázisainak megtekintését.',
+        'database_update' => 'Engedélyezi a felhasználó számára, hogy módosításokat hajtson végre az adatbázison. Ha a felhasználónak nincs "Jelszó megtekintése" engedélye, akkor nem tudja módosítani a jelszót.',
+        'database_delete' => 'Engedélyezi a felhasználó számára az adatbázis(ok) törlését.',
+        'database_view_password' => 'Engedélyezi a felhasználó számára az adatbázis-jelszó megtekintését a rendszerben.',
+        'schedule_create' => 'Lehetővé teszi a felhasználó számára, hogy új időbeosztást hozzon létre a szerverhez.',
+        'schedule_read' => 'Engedélyezi a felhasználó számára a szerver időbeosztásainak megtekintését.',
+        'schedule_update' => 'Engedélyezi a felhasználó számára, hogy módosítsa a meglévő szerver időbeosztásait.',
+        'schedule_delete' => 'Lehetővé teszi a felhasználó számára, hogy törölje a szerver időbeosztását.',
```

Comment:
itt inkább ütezemett feladatot (vagy valami hasonlót) jelent a schedule.

```suggestion
        'schedule_create' => 'Lehetővé teszi a felhasználó számára, hogy új ütemezett feladatot hozzon létre a szerverhez.',
        'schedule_read' => 'Engedélyezi a felhasználó számára a szerver ütemezett feladatainak megtekintését.',
        'schedule_update' => 'Engedélyezi a felhasználó számára, hogy módosítsa a meglévő szerver ütemezett feladatait.',
        'schedule_delete' => 'Lehetővé teszi a felhasználó számára, hogy törölje a szerver ütemezett feladatait.',
```